test/test_evaluator.cpp

```diff
@@ -535,4 +538,24 @@ TEST_F(ExprEvalTest, TernaryOps) {
   }
 }
 
+TEST_F(ExprEvalTest, Permute) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  TensorView* in =
+      TensorViewBuilder().shape({-1, 6}).dtype(DataType::Float).build();
```

Comment:
Could you test a higher dimensional tensor? For example, a tensor whose shape is `(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)`, by using higher dimensional tensor, it will be more likely to find potential bugs in the implementation.

Reply:
Done. It's testing 4D now.